### PR TITLE
Fix Streamembed

### PR DESCRIPTION
**Pull request by DWH-WFC**
regex fixed

### CODE REVIEW

File: script.module.resolveurl/lib/resolveurl/plugins/streamembed.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def get_media_url(self, host, media_id):
                 host, data.group(1), data.group(2)
             )
             html = self.net.http_GET(url, headers=headers).content
-            sources = re.findall(r'[A-Z]{10}=\d+x(?P<label>[\d]+)\s*(?P<url>[^\n]+)', html)
+            sources = re.findall(r'[A-Z]{10}=\d+x(?P<label>[\d]+).*?[\s*|h](?P<url>[^\n]+)', html)
```
Comment (Gujal00) [inline script.module.resolveurl/lib/resolveurl/plugins/streamembed.py:41]: not sure if this is right `[\s*|h\` usually it is a list of characters so [\sh] makes sense

**Comment by Gujal00** [comment]
@DWH-WFC I dont see a need for this PR as the embeds I have work correctly. Can you provide the embed for which this PR is required. Thanks

**Comment by kasi45** [comment]
Hi,

>  Can you provide the embed for which this PR is required. Thanks

https://mp4player.site/watch?v=SCKQLI8M

Unbenannt

**Comment by Gujal00** [comment]
@kasi45 Thanks for the embed, the regex change does not solve the issue as fundamentally that step should not be performed on this type of hls file where audio and video streams are separated. The above PR will play but only the video stream and no audio, so not good.

**Comment by kasi45** [comment]
everything is fine
Thank you very much

**Pull request closed without merging**